lib/config.ts

```diff
@@ -6,7 +6,6 @@ export class Configuration extends ConfigBase implements IConfiguration { // Use
 	CI_LOGGER = false;
 	DEBUG = false;
 	TYPESCRIPT_COMPILER_OPTIONS = {};
-	USE_PROXY = false;
```

Comment:
> The functionality is introduced as command, so there's no need to manually edit any file. 

I ment will we still have a way to enable/disable the proxy in any way through the newly created command.